### ISSUE TITLE
Migrate to QgsSettings?

### ISSUE THREAD

**Issue by elpaso** [north-road/inasafe#10]
this is not technically required.

**Comment by nyalldawson** [comment]
Is there any benefit for plugins with this change?

**Comment by elpaso** [comment]
No, but maybe useful in the future if inasafe is deployed in large orgs and they want to ship some pre-configured settings through the global settings file.

I would say this is very low priority.

**Comment by nyalldawson** [comment]
Ok, let's tag this as a possible, and revisit if the remaining budget after the main port allows.
